src/troute-routing/troute/routing/diffusive_utils.py

```diff
@@ -541,15 +625,17 @@ def diffusive_input_data_v02(
     -------
     diff_ins -- (dict) formatted inputs for diffusive wave model
     """
-    
+
     # lateral inflow timestep (sec)
     dt_ql_g = dt * qts_subdivisions
     # upstream boundary condition timestep (sec)
     dt_ub_g = dt
     # downstream boundary condition timestep (sec)
-    dt_db_g = dt * qts_subdivisions
+    dt_db_g = 360.0 # dt * qts_subdivisions
     # tributary inflow timestep (sec)
     dt_qtrib_g = dt
+    # usgs_df time step used for data assimilation. The original timestep of USGS streamflow is 15 min but later interpolated at every dt in min.
+    dt_da_g = dt # [sec]
```

Comment:
Do we need a new parameter for this, or can we just operate under the assumption that the USGS data will be at time resolution dt?